specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-07-01/virtualMachine.json

```diff
@@ -1326,7 +1326,7 @@
           "VirtualMachines"
         ],
         "operationId": "VirtualMachines_PowerOff",
-        "description": "The operation to power off (stop) a virtual machine. The virtual machine can be restarted with the same provisioned resources. You are still charged for this virtual machine.",
+        "description": "The operation to power off (stop) a virtual machine. The virtual machine can be restarted with the same provisioned resources. You are still charged for this virtual machine. NOTE: This operation is not allowed on a deallocated virtual machine.",
```

Comment:
This operation is not allowed on a virtual machine that is being deallocated or has already been deallocated.